package.json

```
@@ -29,25 +30,35 @@
     "url": "https://github.com/readmeio/oas-normalize/issues"
   },
   "scripts": {
-    "lint": "eslint .",
+    "build": "tsc",
+    "lint": "eslint . --ext .js,.ts",
+    "prebuild": "rm -rf dist/",
+    "prepack": "npm run build",
     "pretest": "npm run lint",
-    "prettier": "prettier --list-different --write \"./**/**.{js,jsx}\"",
-    "test": "jest --coverage"
+    "prettier": "prettier --list-different --write \"./**/**.{js,ts}\"",
```

Wanna add a Markdown check too and then add it to CI?
```suggestion
    "prettier": "prettier --list-different \"./**/**.{md,js,ts}\"",
    "prettier:write": "prettier --list-different --write \"./**/**.{md,js,ts}\"",
```